content/en/cloud_cost_management/custom_allocation_rules/_index.md

```diff
@@ -0,0 +1,85 @@
+
+You can also specify how cost proportions should be partitioned to ensure segment-specific allocations. For example, if you partition your costs by `environment` using tags like `staging` and `production`, the proportions are calculated separately for each environment. This ensures allocations are based on the specific proportions within each partition.
+
+{{< img src="cloud_cost/custom_allocation_rules/proportional_partition_diagram.png" alt="Diagram illustrating the proportional split strategy with partitioning" style="width:90%;" >}}
+
+{{% /tab %}}
+
+{{% tab "Even Allocation" %}}
+
+With the even strategy, costs are allocated evenly towards your destination tags, regardless of any other spend. Similarly to proportional allocation, you can further customize your allocation by setting filters and partitions.
+
+{{< img src="cloud_cost/custom_allocation_rules/even_diagram.png" alt="Diagram illustrating the even split strategy" style="width:60%;" >}}
+
+{{< img src="cloud_cost/custom_allocation_rules/even_ui.png" alt="The even split strategy as seen in Datadog" style="width:60%;" >}}
+
+{{% /tab %}}
+
+{{% tab "Custom Percentage Allocation" %}}
+With the custom percentage strategy, you can define static custom percentages for the destination tags you select. For example, if you have 3 destinations (`teamA`, `teamB`, `teamC`) you can allocate 60% to `teamA`, 30% to `teamB`, and 10% to `teamC`.
+
+{{< img src="cloud_cost/custom_allocation_rules/custom_percentage_diagram.png" alt="Diagram illustrating the even split strategy" style="width:60%;" >}}
+
+{{< img src="cloud_cost/custom_allocation_rules/custom_percentage_ui.png" alt="The even split strategy as seen in Datadog" style="width:60%;" >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Specify what costs are included in the allocation
+| Step | Required | Examples |
+| ---- | ---- | ---- |
+| Cost Provider | Yes | AWS, GCP, Azure |
+| Define the costs to split (source) | Yes | `aws_product` is `support` `allocated_spend_type` is untagged |
+| Define the destination | Yes | `team` is `teamA`, `teamB`, `teamC` |
+| Filter by | Only applicable for Proportional and Even strategies, optional | `aws_product` is `ec2` |
+| Partition costs by | Only applicable for Proportional and Even strategies, optional | `environment` is all values |
+| Rule Name | Yes | allocate\_untagged\_support\_costs |
+
+## Managing rules
+Rules can be modified and deleted in the Custom Allocation Rules section of the Cloud Cost settings page. All fields except for the rule name can be reconfigured.
+
+Rules are applied in the same order as shown in the list.
+
+## Visualize your allocations
+Changes to dynamic allocation rules may take up to 24 hours to be applied. Once applied, the new allocations can be seen throughout the Cloud Cost product. Dynamically allocated costs also include an `allocated_by_rule` tag, denoting the rule name that applied the allocation.
```

Comment:
```suggestion
Changes to dynamic allocation rules may take up to 24 hours to be applied. After being applied, the new allocations can be seen throughout Cloud Cost Management. Dynamically allocated costs also include an `allocated_by_rule` tag, denoting the rule name that applied the allocation.
```

Reply:
Per the [documentation style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#words-and-phrases)